instrumentation/jodd-http-4.2/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/joddhttp/v4_2/JoddHttpHelper.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.joddhttp.v4_2;
+
+import static io.opentelemetry.javaagent.instrumentation.joddhttp.v4_2.JoddHttpSingletons.instrumenter;
+
+import io.opentelemetry.context.Context;
+import jodd.http.HttpRequest;
+import jodd.http.HttpResponse;
+
+public class JoddHttpHelper {
+
+  public static void doMethodExit(
+      Context context, HttpRequest request, Object result, Throwable throwable) {
+    if (throwable != null) {
+      instrumenter().end(context, request, null, throwable);
+    } else if (result instanceof HttpResponse) {
+      instrumenter().end(context, request, (HttpResponse) result, null);
+    }
```

Comment:
This static class isn't doing much. Does it provide value somehow being separate, versus just inlining it? Seems like other similar instrumentations might be doing something similar...wonder why?

Reply:
As mentioned, I orientated myself on the `apache-httpclient-5.0` which also has a `ApacheHttpClientHelper`. Its only used in the `ApacheHttpClientInstrumentation` multiple times, I thought maybe it was some kind of a convention or else I dont know why it's done like that.

In my case the `JoddHttpHelper` is only used in the instrumentation class once, so as you suggested I removed the helper class and inlined the code